panels/class-debug-bar-queries.php

```diff
@@ -79,13 +87,19 @@ function render() {
 			$this->render_panel_info_block( __( 'Total Queries:', 'debug-bar' ), number_format_i18n( $wpdb->num_queries ) );
 		}
 		if ( $total_time ) {
-			/* translators: %0.1f = duration in microseconds. */
-			$duration = sprintf( __( '%0.1f ms', 'debug-bar' ), number_format_i18n( ( $total_time * 1000 ), 1 ) );
-			$this->render_panel_info_block( __( 'Total query time:', 'debug-bar' ), $duration );
+			/* translators: %s = duration in milliseconds. */
+			$this->render_panel_info_block( __( 'Total query time:', 'debug-bar' ), sprintf( __( '%s ms', 'debug-bar' ), number_format_i18n( ( $total_time * 1000 ), 1 ) ) );
 		}
 		if ( ! empty( $EZSQL_ERROR ) ) {
 			$this->render_panel_info_block( __( 'Total DB Errors:', 'debug-bar' ), count( $EZSQL_ERROR ) );
 		}
+		if ( $wpdb->num_queries ) {
+			$sortby .= '<select name="wpd-queries-sort-by" id="wpd-queries-sort-by">';
+			$sortby .= '<option value="elapsed">' . __( 'Elapsed', 'debug-bar' ) . '</option>';
+			$sortby .= '<option value="ordinal" selected="selected">' . __( 'Execution Order', 'debug-bar' ) . '</option>';
```

Comment:
From a usability point of view, it is counter-intuitive to have the default selection be the last in the list instead of the first.